src/main/java/com/faforever/client/teammatchmaking/TeamMatchmakingController.java

```diff
@@ -43,25 +58,68 @@
   @FXML
   public StackPane teamMatchmakingRoot;
   @FXML
-  public JFXListView<PartyMember> playerListView;
-  @FXML
   public JFXButton leavePartyButton;
   @FXML
   public JFXButton readyButton;
   @FXML
   public Label refreshingLabel;
-
+  public ToggleButton uefButton;
+  public ToggleButton cybranButton;
+  public ToggleButton aeonButton;
+  public ToggleButton seraphimButton;
+  @FXML
+  public ImageView avatarImageView;
+  @FXML
+  public ImageView countryImageView;
   @FXML
-  public JFXListView<MatchmakingQueue> queueListView;
+  public Label clanLabel;
+  @FXML
+  public Label usernameLabel;
+  @FXML
+  public Label teamRatingLabel;
+  @FXML
+  public Label gameCountLabel;
+  public Label ladderRatingLabel;
+  public HBox queueBox;
+  public PartyMemberItemController controller;
```

Comment:
what controller?

Reply:
Just realized this line is never used. Must be slipped in by mistake.